metau-capstone/View Controllers/StudyViewController.m

```diff
@@ -321,4 +322,40 @@ - (IBAction)didTapLogout:(id)sender {
     [utility logout: sceneDelegate];
 }
 
+- (void) viewWillAppear:(BOOL)animated {
+    PFUser *const user = [PFUser currentUser];
+    PFQuery *query = [PFUser query];
+    [query getObjectInBackgroundWithId:user.objectId
```

Comment:
Is this outer query necessary? can you rely only on the `queryForCards` to determine whether the user exist & the cards exist for this user? based on my understanding of query, the outer is not necessary for your purpose. just want to confirm

Reply:
I call the first line because I use user.objectID in the second query. I want to query for the current user's cards.